include/openssl/err.h

```diff
@@ -225,7 +238,6 @@ struct err_state_st {
 # define ERR_R_MISSING_ASN1_EOS                  63
 
 /* fatal error */
-# define ERR_R_FATAL                             64
```

Comment:
This was wrongly moved

Reply:
I disagree, because it's part of the composition of an error code, so it belongs with the other func/lib/reason defines.

Reply:
It's part of the composition of *reason codes* (hence `_R_`)

Reply:
It is also a part of the partitioning of the error-code space, so it should be with the other shifts and masks that define the partitioning.  It's a matter of taste, I guess.